API_usage.md

```diff
@@ -5,14 +5,14 @@ Scientist publish the data to this STAC catalog to make it available to the user
 
 Follow the guide below to publish datasets to the VEDA STAC catalog.
 
-## Prepare the data
+## STEP I: Prepare the data
 
 VEDA supports inclusion of cloud optimized GeoTIFFs (COGs) to its data store.
 
 ### Creating COGs
```

Comment:
We have a more elaborate section on Creating COGs in the veda-workflows-api docs: https://github.com/NASA-IMPACT/veda-workflows-api/blob/main/how-to.md#creating-cogs

How about copying that here?

Reply:
oh that's a great idea, i have forgotten that it existed! i'll do that